python/lsst/ts/xml/data/sal_interfaces/MTMount/MTMount_Events.xml

```diff
@@ -64,12 +64,17 @@
         System_OilSupplySystem=9,
         System_AzimuthDrivesThermal=10,
         System_ElevationDrivesThermal=11,
-        System_AZ0101CabinetThermal=12,
-        System_ModbusTemperatureControllers=13,
+        System_Cabinet0101Thermal=12,
+        System_AuxiliaryCabinetsThermal=13,
         System_MainCabinetThermal=14,
         System_MainAxesPowerSupply=15,
         System_TopEndChiller=16
     </Enumeration>
+  <Enumeration>
+      ThermalCommandState_NoChange = 0,
+      ThermalCommandState_Off = 1,
+      ThermalCommandState_On = 2
```

Comment:
In MTMount_Commands.xml you menation a `Fault` state but I don't see that here. Shouldn't that be added as well?

Reply:
The actual state can, of course be Fault, but those are the PowerState enum values.

The ThermalCommandState enums are the _commanded_ state of the thermal control system and these are the only three options: leave it as is, turn it off, or turn it on. You cannot command any MTMount subsystem (or, indeed, most of our systems -- the only exception I know of is MTRotator) to fault state.